azerothcore/azerothcore-wotlk
# fix(Core/Spell): Clicking lock portal cost mana

Remove the Percentage cost of mana from the spell that trigger if you click the portal





## CHANGES PROPOSED:
-  Remove the percentage cost from the spell that trigger when you click on the warlock portal.


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3856



## TESTS PERFORMED:
- builds
- tested in-game



## HOW TO TEST THE CHANGES:

1. Go in a group with 3 people (1 lock required ofc)
2. learn 698 (Ritual of Summoning)
3. Go below 12% mana (Easy with `.learn all my spells`, `.set mana 1195` and cast Demon Skin)
4. Try to summon a player and note that it does not say "You don't have enough mana"

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
